src/api_server/src/token/simple.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2023 by Alibaba.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::*;
+use jwt_simple::algorithms::{ECDSAP256KeyPairLike, ECDSAP256PublicKeyLike};
+use jwt_simple::prelude::{Claims, Duration, ES256KeyPair};
+use rcgen::{Certificate, CertificateParams};
+use serde_json::Value;
+
+use crate::token::TokenBroker;
+
+pub struct SimpleTokenBroker {
+    key_pair: ES256KeyPair,
+}
+
+impl SimpleTokenBroker {
+    pub fn new() -> Result<Self> {
+        let key_pair = ES256KeyPair::generate();
+        Ok(Self { key_pair })
+    }
+}
+
+impl TokenBroker for SimpleTokenBroker {
+    fn issue_attestation_token(&self, custom_claims: Value) -> Result<String> {
+        let claims = Claims::with_custom_claims(custom_claims, Duration::from_mins(20));
+        let token = self.key_pair.sign(claims)?;
+        Ok(token)
+    }
+
+    fn verify_attestation_token(&self, token: String) -> Result<()> {
+        let _ = self
+            .key_pair
+            .public_key()
+            .verify_token::<Value>(&token, None)?;
+        Ok(())
+    }
+
+    fn get_x509_certificate(&self) -> Result<Vec<u8>> {
+        let rcgen_key_pair = rcgen::KeyPair::from_pem(&self.key_pair.to_pem()?)?;
+        let mut cert_params = CertificateParams::new(vec!["CoCo-Attestation-Service".to_string()]);
+        cert_params.key_pair = Some(rcgen_key_pair);
+        let cert = Certificate::from_params(cert_params)?.serialize_der()?;
+        Ok(cert)
```

Comment:
We may want to cache that certificat and avoid having to regenerate the same one for each call?